2020/prep/filter_benchmarks.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+filter_benchmark () {
```

Comment:
Could you add documentation to what the script is doing?

Reply:
done.